ipad/Constants/StringConstants.swift

```diff
@@ -69,9 +69,11 @@ struct ShiftFormHeaders {
     struct ShiftStart {
         static let startTime = "Shift Start Time *"
         static let k9OnShift = "k9 On Shift"
-        static let station = "Station"
+        static let station = "Station *"
```

Comment:
Selecting a station is validated (needs to be selected) so I added the asterisk here to let the user know